### DISCUSSION TITLE
Excited AND frustrated....

### DISCUSSION THREAD

**Discussion by blender50** [StephenMiracle/frankenwp#26] [Q&A]
TLDR; Where is the documentation and working slack invite?

I recently stumbled upon wpeverywhere and am excited for the potential. 

However when I go to https://wpeverywhere.com/  and https://wpeverywhere.com/introducing-the-new-wpeverywhere-docker-hub-registry/ there is a mention made of "Our comprehensive documentation will guide you through the setup process, helping you deploy your WordPress site in minutes.
" I am at a loss to find said documentation other than the README.md. Also there is an invite to join a slack channel that is expired and doesn't work.

I understand that this is comparatively new and as such will be going through some growing pains. Also my use case may be a bit of a corner case, but without hearing from other users this may not be the case? (shrug)

If I am able to smooth things out by either seeing more documentation, or direct contact through slack, I am not opposed to contributing to the documentation effort in some way if that helps.

**Comment by sveken** [comment]
ANy update on this? i am stuck getting redirected to :80 behind my cloudflare tunnel.

**Comment by blender50** [comment]
Sadly no.

**Comment by nickchomey** [comment]
Can I suggest you rewrite your post? Perhaps even delete this one altogether. I'm sure you have good intentions, but it all comes across VERY poorly - entitled, impatient, incredulous etc... Just state your enthusiasm and what specifically and clearly you'd like to see in order to be able to better use, and perhaps even contribute to, this. This, and most of open source, is done in spare time and generosity.
